ql/src/java/org/apache/hadoop/hive/ql/exec/tez/TezTask.java

```diff
@@ -361,7 +361,7 @@ private void collectCommitInformation(TezWork work) throws IOException, TezExcep
           .filter(name -> name.endsWith("HiveIcebergNoJobCommitter")).isPresent();
       // we should only consider jobs with Iceberg output committer and a data sink
       if (hasIcebergCommitter && !vertex.getDataSinks().isEmpty()) {
-        String tableLocationRoot = jobConf.get("location");
+        String tableLocationRoot = jobConf.get("iceberg.mr.table.location");
```

Comment:
What is this change?
Shall we use a constant?

Reply:
Could use a constant, sure. The table location is not available at "location" key anymore, but at this one "iceberg.mr.table.location"